examples/nodes.txt

```diff
@@ -0,0 +1,6 @@
+id	time	is_sample
+0	0.0	1
+1	0.0	1
+2	1.0	1
```

Comment:
Shouldn't 2,3 and 4 be non-samples? This is causing the trees to print out in a funny way below.

Reply:
I had tried that initially but it didn't draw the tree (2 (0, 1)): 97cc3b91d0be83f91b86078696371837e676cd12

Reply:
Er, never mind I made a mistake here

Reply:
It is nice that if you mark them all as samples you can see the variation in tree height depicted in fig5c